app/views/hub/clients/index.html.erb

```diff
@@ -2,13 +2,23 @@
 
 <% content_for :card do %>
   <section class="slab slab--padded client-index-header">
-    <%= render 'shared/user_header' %>
+    <%= render "shared/user_header" %>
     <%= render "shared/dashboard_navigation" %>
     <%= render "shared/client_filters" %>
 
     <div class="actions-wrapper">
       <%= link_to t(".add_client"), new_hub_client_path, class: "button" %>
     </div>
+
+    <%= page_entries_info @clients, model: "entry" %>
+    <%= will_paginate(
+          @clients,
+          previous_label: "<i class=\"icon icon-keyboard_arrow_left\"></i><span class=\"hide-on-mobile\"> Previous</span>",
```

Comment:
seems a little verbose that we're having to pass in the exact icons as a string. would love to see a partial or helper for the icon/label so we can easily reuse, or better yet a wrapper around the will_paginate helper that creates a consistent styling that we can add to other pages. But as we're only using in 2 places, doesn't violate the rule of threes to leave this refactor for another time!

Reply:
This is a great point! I plan to follow up soon with a styling commit, which would be a good opportunity to address this.